pyprocessing/color.py

```diff
@@ -4,25 +4,40 @@
 
 
 class Color:
-    RGB = 0
-    HSV = 1
+    RGB = 1
     HLS = 2
+    HSB = 3
 
-    def __init__(self, *args, colorspace=0):
-        self.colorspace = colorspace
+    colorspace = RGB
+    maxv1 = maxv2 = maxv3 = maxva = 255
+
+    def __init__(self, *args):
+        def adjust(val, max):
+            if val > max:
+                return 255
+            return round((val / max) * 255)
```

Comment:
cast into an int, don't use max as a variable name, as it's also a python builtin.

Built-ins should never be overridden unless they share a common name with processing's built-ins (i.e. exit), even when the scope would pose no issues.